packages/expo-application/src/Application.ts

```diff
@@ -49,6 +57,16 @@ export const applicationId: string | null = ExpoApplication
   ? ExpoApplication.applicationId || null
   : null;
 
+// @needsAudit
+/**
+ * Returns the compiled Apple `Info.plist` for the app. This function is `undefined` on Android and web.
```

Comment:
What do you mean by "compiled" here?

Reply:
The compiled Info.plist contains build settings that aren't shown in the local Info.plist file.

Reply:
OK, would be clearer to say:
```suggestion
 * Returns an object containing the fields of the Apple `Info.plist` configuration file compiled into this build of the app.
 * This function is `undefined` on Android and web.
```